main.js

```diff
@@ -0,0 +1,23 @@
+let cnt = 0
+document.getElementById('listen').addEventListener('click',() => {
+    if(localStorage.getItem('counter'))
```

Comment:
let's cache this variable if we are using it multiple times, better yet move the getters and setters of local storage to a diff function altogehter